dask_geopandas/core.py

```diff
@@ -389,6 +389,60 @@ def morton_distance(self, p=15):
 
         return distances
 
+    def shuffle(self, by="hilbert", column=None, drop=True, npartitions=10, **kwargs):
```

Comment:
It needs to be called differently (`spatial_shuffle`?), dask already has the `shuffle` method.